src/@next/Tabs/TabStyle.ts

```diff
@@ -0,0 +1,96 @@
+import styled from 'styled-components';
+import { Blue, Neutral } from '../utilities/colors';
+import { space16, space4, space8 } from '../utilities/spacing';
+import * as Breakpoints from '../utilities/breakpoints';
+
+export const StyledTabsContainer = styled.div`
+  display: block;
+`;
+
+export const StyledTabHeaderContainer = styled.div`
+  box-shadow: inset 0px -1px 0px #e1e3e5;
+`;
+
+export const StyledUl = styled.ul`
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 0;
+  gap: ${space8};
+
+  &[data-fitted='true'] li {
+    flex: 1;
+  }
+`;
+
+export const StyledLi = styled.li`
+  display: flex;
+  margin: 0;
+  padding: 0 ${space4};
+`;
+export const StyledSpan = styled.span`
+  position: relative;
+  display: block;
+  padding: 0 ${space16};
+
+  &::before {
+    content: '';
+    position: absolute;
+    bottom: -${space16};
+    left: 0;
+    right: 0;
+    height: 3px;
+    border-top-left-radius: ${space4};
+    border-top-right-radius: ${space4};
+  }
+
+  &&::after {
+    content: '';
+    position: absolute;
+    z-index: 1;
+    top: -1px;
+    right: -1px;
+    bottom: -1px;
+    left: -1px;
```

Comment:
able to use shorthand `inset: -1px` ?

Reply:
Oh didn't know it has a shorthand, thanks! updated